workspaces/ui-v2/src/styles/stylingHooks.ts

```diff
@@ -0,0 +1,49 @@
+import { darken, makeStyles, Theme } from '@material-ui/core';
+
+type SpacingInputs = {
+  size?: number;
+};
+
+export const useSpacingStyles = makeStyles<Theme, SpacingInputs>((theme) => ({
```

Comment:
I added these useSpacingStyles but I don't think I like this. I would think the general usecases here are that we'd want margin + padding to be _different_ (i.e. margin top and bottom with size 2 and left and right with size 1, with other padding stuff).

I actually don't like the material ui makeStyles, especially for ad-hoc usages with styling. My preference is to use predefined css classes - e.g. `mt-l mb-l` which correspond to `marginTop: large -> 24px` etc where we have different preset spacing units.